spec/services/creneaux_search/calculator/busy_time_preloader_spec.rb

```diff
@@ -118,4 +118,30 @@
       end
     end
   end
+
+  describe "request to fetch rdvs" do
+    it "est optimisée pour utiliser l'index 'calculator_index'. Décommentez le test suivant si celui-ci échoue." do
+      # Voir https://www.postgresql.org/docs/current/indexes-index-only-scans.html
+      request = described_class.new(range, plage_ouverture).send(:optimized_rdv_request)
+      expect(request.select(:calculator_rdv_starts_at, :calculator_rdv_ends_at).to_sql).to eq <<~SQL.squish
+        SELECT "agents_rdvs"."calculator_rdv_starts_at", "agents_rdvs"."calculator_rdv_ends_at" FROM..._starts_at, calculator_rdv_ends_at, '[)') && tsrange('2021-10-26 06:00:00', '2021-10-29 10:00:00'))
+      SQL
```

Comment:
Ça échouait chez moi à cause du tronquage (`...`) qui est juste un affichage de RSpec et de l'ID de l'agent qui varie.

Je propose la version ci-dessous. ;)

```suggestion
      expect(request.select(:calculator_rdv_starts_at, :calculator_rdv_ends_at).to_sql.squish).to eq <<~SQL.squish
        SELECT "agents_rdvs"."calculator_rdv_starts_at",
               "agents_rdvs"."calculator_rdv_ends_at"
        FROM "agents_rdvs"
        WHERE "agents_rdvs"."agent_id" = #{plage_ouverture.agent_id}
          AND "agents_rdvs"."calculator_rdv_not_cancelled_and_in_the_future" = TRUE
          AND (tsrange(calculator_rdv_starts_at, calculator_rdv_ends_at, '[)') && tsrange('2021-10-26 06:00:00', '2021-10-29 10:00:00'))
      SQL
```

Reply:
haha, clairement je suis allé trop vite en committant ça à l'arrache hier soir. Merci pour le fix !